src/DaxStudio.UI/Views/AutoSaveRecoverDialogView.xaml

```diff
@@ -94,7 +94,7 @@
                       cal:Message.Attach="[Event MouseUp] = [Action ToggleShouldOpen($this.SelectedItem)]">
             </ListView>
             <StackPanel Orientation="Horizontal" Grid.Row="3" Margin="7,5" HorizontalAlignment="Right">
-                <Button x:Name="Open">Open</Button>
+                <Button x:Name="_Open" IsDefault="True">Open</Button>
```

Comment:
I think you've accidentally changed the x:Name property instead of changing the label. This will break the Caliburn.Micro binding to the action. If you want to fix this I can then merge this pull request

Reply:
I think I've fixed it...  TBH I am pretty new with GitHub PR's -- I just added another commit.  If you merge, does it merge the last version of my change?

Reply:
Yes, I'm still figuring out how PRs work myself. But it does  merge the latest version of your change, so I can see that the underscore is now in the correct place